unison-cli/src/Unison/Codebase/Editor/HandleInput.hs

```diff
@@ -2000,6 +1945,81 @@ handleShowDefinition outputLoc inputQuery = do
             Nothing -> Just "scratch.u"
             Just (path, _) -> Just path
 
+-- | Handle a @test@ command.
+handleTest :: Monad m => TestInput -> Action' m v ()
```

Comment:
In the future when refactoring into a method like this it's really helpful if you can point out what changed in the PR, (or if nothing changed) since the diff is just all green I don't know what to look for. 😄

Reply:
I did.

Reply:
Oh weird, it didn't show in the diff editor 🤔 ; my bad! Or rather, Github's bad

Reply:
😁